### PR TITLE
Split and cleaned up JS files.

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides/agenda/static/js/agenda/base.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+(function () {
```
Comment (ostcar) [inline openslides/agenda/static/js/agenda/base.js]: Why do you added this closure? We work with angular modules which do the same thing. I don't think that we need double closures.

Comment (normanjaeckel) [inline openslides/agenda/static/js/agenda/base.js]: We do not always just call an angular function. Sometimes there are extra functions defined in a file. I would say using a closure is always best practice to protect the global namespace.

Comment (ostcar) [inline openslides/agenda/static/js/agenda/base.js]: This other function should be coded as angular services.

I don't see the advantage to use two techincs to solve one problem.

**Comment by normanjaeckel** [comment]
Anything important open here? So we can merge this.

**Pull request merged**